site/en/posts/chromeos-112-113-release-notes.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 'ChromeOS 112 and 113 improve Screen Capture, screensavers, and security'
+metadesc: 'The latest ChromeOS release includes updates to Screen Capture, screensavers, and security.'
```

Comment:
Can we also normalize the descriptions? Some of these are directly descriptive, like 116, where some mention the current version (here, 114)